### ISSUE TITLE
Missing keys in default config file

### ISSUE THREAD

**Issue by klemens-st** [rjsears/chia_plot_manager#50]
To reproduce:
* follow the instructions and install with `install.sh`
* try `python3 drive_manager.py`

We get this:
```
Traceback (most recent call last):
  File "drive_manager.py", line 115, in <module>
    chianas = DriveManager.read_configs()
  File "/root/plot_manager/drivemanager_classes.py", line 115, in read_configs
    total_plot_highwater_warning=server['harvester']['total_plot_highwater_warning'],
```
and if we rectify the problem by adding the missing key, this follows:
```
Traceback (most recent call last):
  File "drive_manager.py", line 115, in <module>
    chianas = DriveManager.read_configs()
  File "/root/plot_manager/drivemanager_classes.py", line 116, in read_configs
    total_plots_alert_sent=server['harvester']['total_plots_alert_sent'],
KeyError: 'total_plots_alert_sent'
```

This is because `total_plot_highwater_warning` and `total_plots_alert_sent` are directly accessed in `drivemanager_classes.py` but not present in the default config file.

**Comment by klemens-st** [comment]
I almost forgot: thanks for this very useful module 🎆 I was going to submit a PR instead of this issue but alas running short on time.

**Comment by rjsears** [comment]
Thank you for pointing this out, I have updated the skel file! If you reclone and rerun the install it should fix the issue. It should keep all of your current config info (if I have not messed something up) but it will make a backup copy of it before it does anything. If you are running the standard code you can simply move your current directory to a backup spot and reclone and reinstall.

**Comment by klemens-st** [comment]
Thanks, I fixed it earlier by simply adding missing data to the config so no need to reclone. Just wanted to point it out in case others stumble upon it. 

By the way, are you planning, or maybe have uncommited code, to add support for multiple plotters? If not I will probably code it myself at some point, right now I simply created a second harvester instance on Proxmox to receive plots from my second plotter. I guess I will have to remove the bit that kills all remote nc processes but here I have a question for you: why did you put it there in the first place? wasn't it enough to just kill the listener that was actually being used? were there other bugs that made you do that?
